docs/Datadog.Trace/README.md

```diff
@@ -0,0 +1,229 @@
+
+```csharp
+using Datadog.Trace;
+
+// No changes required here
+using (var scope = Tracer.Instance.StartActive("my-operation")
+{
+}
+
+// No Longer compiles (incorrect usage of Scope)
+using (Scope scope = Tracer.Instance.StartActive("my-operation")
+{
+}
+
+// Correct usage with explicit type
+using (IScope scope = Tracer.Instance.StartActive("my-operation")
+{
+}
+```
+
+### Simplification of the tracer interface
+
+In addition to returning `IScope`, the `Tracer.StartActive` method has been simplified to provide a `SpanCreationSettings`. In addition, we have removed the possibility to override the service name.
+
+
+### Incorrect integration names are ignored
+
+In version 2.0, any changes made to an `IntegrationSettings` object for an unknown `IntegrationId` will not be persisted. Instead, a warning will be logged describing the invalid access.
+
+```csharp
+using Datadog.Trace;
+
+var tracerSettings = TracerSettings.FromDefaultSources();
+
+// Accessing settings for an unknown integration will log a warning
+var settings = tracerSettings .Integrations["MyRandomIntegration"];
+
+// changes are not persisted
+settings.Enabled = false; 
+
+// isEnabled is null, not false
+bool? isEnabled = tracerSettings.Integrations["MyRandomIntegration"].Enabled;
+```
+
+### Automatic instrumentation changes
+
+#### Default value of `DD_TRACE_ROUTE_TEMPLATE_RESOURCE_NAMES_ENABLED` is now `true`
+
+Version 1.26.0 added support for the `DD_TRACE_ROUTE_TEMPLATE_RESOURCE_NAMES_ENABLED` feature flag, which enabled improved span names for ASP.NET and ASP.NET Core automatic instrumentation spans, an additional span for ASP.NET Core requests, and additional tags.
+
+In version 2.0, `DD_TRACE_ROUTE_TEMPLATE_RESOURCE_NAMES_ENABLED` is **enabled** by default. Due to the change in span names, you may need to update your monitors and dashboards to use the new resource names.
+
+If you do not wish to take advantage of the improved route names, you can disabled the feature by setting the `DD_TRACE_ROUTE_TEMPLATE_RESOURCE_NAMES_ENABLED` environment variable to `0`.
+
+#### Call-site instrumentation has been removed
+
+From version 2.0, call-site automatic instrumentation has been removed and replaced with call-target instrumentation. This has been the default mode [since version 1.28.0](https://github.com/DataDog/dd-trace-dotnet/releases/tag/v1.28.0). Call-target instrumentation provides performance improvements over call-site instrumentation, but we no longer support instrumenting some custom implementations of `DbCommand`. If you find spans are missing from your traces after upgrading, please raise an issue on GitHub, or contact [support](https://docs.datadoghq.com/help).
```

Comment:
```suggestion
Call-site automatic instrumentation was removed in .NET Tracer 2.0 and replaced with call-target instrumentation. This was the default mode [since version 1.28.0](https://github.com/DataDog/dd-trace-dotnet/releases/tag/v1.28.0). Call-target instrumentation provides performance and reliability improvements over call-site instrumentation, but it does not support instrumenting custom implementations of `DbCommand` for now. If you find spans are missing from your traces after upgrading, please raise an issue on GitHub, or contact [support](https://docs.datadoghq.com/help).
```

Reply:
The "This" in the second sentence seems a little ambiguous. Maybe just restate "Call-target instrumentation"?